cloudsync_onedrive.py

```diff
@@ -813,6 +813,9 @@ def _convert_to_event(self, change, new_cursor) -> Optional[Event]:
                 return None
 
             path = self._join_parent_reference_path_and_name(parent_path, change['name'])
+            if not path:
+                log.debug("ignoring event: path %s is outside shared folder", parent_path)
```

Comment:
Will this log a lot? It looks like it could, potentially

Reply:
Possibly, in some circumstances. On the other hand, we do log.debug ALL events in the same func above (line 800) so if anything that line should be removed.

I think this log line would be useful in case we end up missing events (ignoring too many).

Reply:
I believe we log ignored events in other locations

Reply:
removed, can infer ignored status from what is already logged.